### PR TITLE
Replace doc links with appropriate ones

### PR DESCRIPTION
**Pull request by ksol**
Replaces a few doc links that are susceptible to pop up. There's at least another one, but I don't know if it's relevant for us (or what would be the appropriate message)

Also, not sure if the right process is to merge this to master, let me know.

### CODE REVIEW

File: lib/language_pack/ruby.rb [hunk 1]
```diff
@@ -460,13 +460,16 @@ def warn_outdated_eol
 
         Your current Ruby version no longer receives security updates from
         Ruby Core and may have serious vulnerabilities. While you will continue
-        to be able to deploy on Heroku with this Ruby version you must upgrade
+        to be able to deploy on Scalingo with this Ruby version you must upgrade
         to a non-EOL version to be eligable to receive support.
```
Comment (EtienneM) [inline lib/language_pack/ruby.rb:464]:
```suggestion
        to a non-EOL version to be eligible to receive support.
```

File: lib/language_pack/ruby.rb [hunk 2]
```diff
@@ -460,13 +460,16 @@ def warn_outdated_eol
...
 
         Upgrade your Ruby version as soon as possible.
 
         For a list of supported Ruby versions see:
-          https://devcenter.heroku.com/articles/ruby-support#supported-runtimes
+          https://doc.scalingo.com/languages/ruby#list-of-the-compatible-runtimes
```
Comment (EtienneM) [inline lib/language_pack/ruby.rb:469]: This link is not the correct one I think.

File: lib/language_pack/ruby.rb [hunk 3]
```diff
@@ -460,13 +460,16 @@ def warn_outdated_eol
...
+
+        For more information on syntax for declaring a Ruby version see:
+          https://doc.scalingo.com/languages/ruby#declaring-version
```
Comment (EtienneM) [inline lib/language_pack/ruby.rb:472]: This link is not the correct one I think.

File: lib/language_pack/ruby.rb [hunk 4]
```diff
@@ -484,7 +487,10 @@ def warn_outdated_eol
         Please upgrade your Ruby version.
 
         For a list of supported Ruby versions see:
-          https://devcenter.heroku.com/articles/ruby-support#supported-runtimes
+          https://doc.scalingo.com/languages/ruby#list-of-the-compatible-runtimes
```
Comment (EtienneM) [inline lib/language_pack/ruby.rb:490]: This link is not the correct one I think.

File: lib/language_pack/ruby.rb [hunk 5]
```diff
@@ -484,7 +487,10 @@ def warn_outdated_eol
...
+
+        For more information on syntax for declaring a Ruby version see:
+          https://doc.scalingo.com/languages/ruby#declaring-version
```
Comment (EtienneM) [inline lib/language_pack/ruby.rb:493]: This link is not the correct one I think.

File: lib/language_pack/ruby.rb [hunk 6]
```diff
@@ -876,7 +885,7 @@ def build_bundler(default_bundle_without)
             error_message += <<-ERROR
 
 Detected sqlite3 gem which is not supported on Scalingo.
-http://doc.scalingo.com/databases/sqlite3
+https://doc.scalingo.com/databases/sqlite3
```
Comment (EtienneM) [inline lib/language_pack/ruby.rb:888]: It's a 404

**Comment by ksol** [comment]
I should have check the links instead of just copying them. thanks

**Pull request merged**